src/test/java/org/opentripplanner/routing/fares/OrcaFareServiceTest.java

```diff
@@ -0,0 +1,209 @@
+            Arguments.of(trip2, Fare.FareType.orcaRegular, 0f + DEFAULT_RIDE_PRICE_IN_CENTS + DEFAULT_RIDE_PRICE_IN_CENTS),
+            Arguments.of(trip2, Fare.FareType.orcaSenior, 0f + DEFAULT_RIDE_PRICE_IN_CENTS + 125f),
+            Arguments.of(trip2, Fare.FareType.orcaYouth, 200f + DEFAULT_RIDE_PRICE_IN_CENTS + 0f),
+            Arguments.of(trip3, Fare.FareType.regular, DEFAULT_RIDE_PRICE_IN_CENTS * 3),
+            Arguments.of(trip3, Fare.FareType.senior, DEFAULT_RIDE_PRICE_IN_CENTS * 3),
+            Arguments.of(trip3, Fare.FareType.youth, DEFAULT_RIDE_PRICE_IN_CENTS * 3),
+            Arguments.of(trip3, Fare.FareType.orcaLift, DEFAULT_RIDE_PRICE_IN_CENTS + 0f + 0f),
+            Arguments.of(trip3, Fare.FareType.orcaRegular, 0f + DEFAULT_RIDE_PRICE_IN_CENTS + 0f),
+            Arguments.of(trip3, Fare.FareType.orcaSenior, 0f + 0f + 200f),
+            Arguments.of(trip3, Fare.FareType.orcaYouth, 0f + 0f + 300f),
+            Arguments.of(trip4, Fare.FareType.regular, DEFAULT_RIDE_PRICE_IN_CENTS * 3),
+            Arguments.of(trip4, Fare.FareType.senior, DEFAULT_RIDE_PRICE_IN_CENTS * 3),
+            Arguments.of(trip4, Fare.FareType.youth, DEFAULT_RIDE_PRICE_IN_CENTS * 3),
+            Arguments.of(trip4, Fare.FareType.orcaLift, 0f + 200f + 0f),
+            Arguments.of(trip4, Fare.FareType.orcaRegular, 0f + DEFAULT_RIDE_PRICE_IN_CENTS + 0f),
+            Arguments.of(trip4, Fare.FareType.orcaSenior, 0f + 0f + 200f),
+            Arguments.of(trip4, Fare.FareType.orcaYouth, 0f + 0f + 300f),
+            Arguments.of(trip5, Fare.FareType.regular, DEFAULT_RIDE_PRICE_IN_CENTS * 2),
+            Arguments.of(trip5, Fare.FareType.senior, DEFAULT_RIDE_PRICE_IN_CENTS * 2),
+            Arguments.of(trip5, Fare.FareType.youth, DEFAULT_RIDE_PRICE_IN_CENTS * 2),
+            Arguments.of(trip5, Fare.FareType.orcaLift, 450f + 0f),
+            Arguments.of(trip5, Fare.FareType.orcaRegular, 575f + 0f),
+            Arguments.of(trip5, Fare.FareType.orcaSenior, 300f + 0f),
+            Arguments.of(trip5, Fare.FareType.orcaYouth, 450f + 0f),
+            Arguments.of(trip6, Fare.FareType.orcaLift, 0f + 0f + 200f),
+            Arguments.of(trip6, Fare.FareType.orcaRegular, 0f + 0f + DEFAULT_RIDE_PRICE_IN_CENTS),
+            Arguments.of(trip6, Fare.FareType.orcaSenior, 0f + 0f + 200f),
+            Arguments.of(trip6, Fare.FareType.orcaYouth, 0f + 0f + 300f)
+        );
+    }
+
+    /**
+     * Build a list of {@link Ride)s from the ride types provided. The values used here to produce a {@link Ride} match
+     * the values used in {@link OrcaFareServiceImpl} to determine the correct ride type.
+     */
+    private static List<Ride> getTrip(OrcaFareServiceImpl.RideType... rideTypes) {
+        List<Ride> rides = new ArrayList<>();
+        for (OrcaFareServiceImpl.RideType rideType : rideTypes) {
+            rides.add(tripStrategy.get(rideType));
+        }
+        return rides;
```

Comment:
Can we use `stream.map(...).collect(Collectors.toList)`?

Reply:
Yep. Updated.